src/backend/core/tests/test_models_invitations.py

```diff
@@ -0,0 +1,46 @@
+"""
+Unit tests for the Invitation model
+"""
+from django.core.exceptions import ValidationError
+
+import pytest
+
+from core import factories
+
+pytestmark = pytest.mark.django_db
+
+
+def test_models_invitations_email_no_empty_mail():
+    """The "email" field is required."""
```

Comment:
Maybe to differentiate from the next test's docstring:
```suggestion
    """The "email" field should not be empty."""
```